contracts/LiquidityManager.sol

```diff
@@ -141,7 +147,8 @@ library LiquidityManager {
                 pending: node.pending,
                 redemptions: node.redemptions.pending,
                 prev: node.prev,
-                next: node.next
+                next: node.next,
+                accrued: node.accrued
```

Comment:
We'll need to return accrualRate and accrualTimestamp as well to calculate this accurately offchain

Reply:
Fixed and updated contract size screenshot in description above